server/actions/repo/scripts/c_script.sh

```diff
@@ -155,7 +170,7 @@ for dir in ${repos[@]}; do
             errorCode=$?
             #Check if the program was terminated
             if [[ $errorCode -eq 142 ]] ; then
-              printf "<p class='alert alert-danger'>Program terminated because of infinite loop.\nPlease run their program manually or check their code.</p>" > $fname.out.html
+              printf "<p class='alert alert-danger'>Program terminated because of infinite loop.\nPlease run their program manually or check their code.</p>" >> $fname.out.html
```

Comment:
Can you keep the old code? The reason why I did this was because if their program had an infinite loop it could have at least 10k lines worth of output and it seems pointless to even try to show the grader the output. Also I don't want to be transferring large files anyway. So just telling the grader there was an infinite loop is enough

Reply:
Ok, I can revert it back, just thought that even if they have runtime errors it would be nice for the tutors to still get the 80 character message